apps/src/templates/projects/projectsRedux.js

```diff
@@ -170,6 +171,7 @@ function projectLists(state = initialProjectListState, action) {
 const initialHasOlderProjects = {
```

Comment:
i see the comment above this object, but i still don't really understand... do you know what this means?

Reply:
If I remember correctly, when we load the public gallery we only fetch 100 published projects of each type. `hasOlderProjects` determines whether there are more projects already on the client to show or more to fetch from the server, which in turn determines whether to hide or show the view more button for the project type. We have soooooooo many projects of most types though that there's a functionally endless supply so I don't know how critical this is.

Reply:
This has a little more context: https://github.com/code-dot-org/code-dot-org/pull/15631 (and interestingly, the feature idea for search in the public gallery)

Reply:
Also because these all default to `true` except `special_topic` which is Hackathon remnants of a feature we never fully implemented, it feels like (outside the context of this PR) we could definitely cleaning both `special_topic` and the `hasOlderProjects` and `initialHasOlderProjects` state up.

Reply:
thank you, that explains it! i'm not sure which cabal would own it, but is there a ticket we can log (or maybe one already exists) for that cleanup?

Reply:
Logged in the LP backlog (https://codedotorg.atlassian.net/browse/LP-2107) but I can't say with any confidence we'll get to it any time soon.